lib/ballot_test.go

```diff
@@ -48,12 +49,7 @@ func TestBallotHash(t *testing.T) {
 
 	nodeRunner := nodeRunners[0]
 
-	// `nodeRunner` is proposer's runner
-	nodeRunner.SetProposerCalculator(SelfProposerCalculator{})
-
-	nodeRunner.Consensus().SetLatestConsensusedBlock(genesisBlock)
```

Comment:
That does not look related ?

Reply:
@Geod24 You're right. I deleted this line because it did not affect the test.

Reply:
Fair enough :)